elrond-config.md

```diff
@@ -188,9 +243,9 @@ TODO: Implement [reserved keys and read-only runtimes](https://github.com/Elrond
     rule <instrs> #isReservedKey(KEY) => . ... </instrs>
       requires notBool #hasPrefix(KEY, "ELROND")
 
-    rule <commands> (. => #exception(UserError)) ... </commands>
-         <instrs> (#isReservedKey(KEY) ~> _) => . </instrs>
-         <message> _ => String2Bytes("cannot write to storage under Elrond reserved key") </message>
+    rule <instrs> #isReservedKey(KEY)
+               => #throwException(UserError, "cannot write to storage under Elrond reserved key") ...
```

Comment:
AFAICT, returned error codes are never used because [`WithFaultAndHost`](https://github.com/multiversx/mx-chain-vm-go/blob/e01b8c0ccac09f8c48d8f811300b44deae0b314e/vmhost/vmhooks/vmHooksImpl.go#L102)  stops the Wasmer instance immediately when there is an error.

* [`StorageStore`](https://github.com/multiversx/mx-chain-vm-go/blob/ea3d78d34c35f7ef9c1a9ea4fce8288608763229/vmhost/vmhooks/baseOps.go#L1649)
* [`WithFaultAndHost`](https://github.com/multiversx/mx-chain-vm-go/blob/e01b8c0ccac09f8c48d8f811300b44deae0b314e/vmhost/vmhooks/vmHooksImpl.go#L102)
* [`runtime.FailExecution`](https://github.com/multiversx/mx-chain-vm-go/blob/ea3d78d34c35f7ef9c1a9ea4fce8288608763229/vmhost/contexts/runtime.go#L577)

Moreover, Rust SDK ignores the function's return code assuming that VM handles the error.
* Declaration: [`fn storageStore(...) -> i32;`](https://github.com/multiversx/mx-sdk-rs/blob/b0160312d19670c81b919f3029858e98f3c2228d/framework/wasm-adapter/src/api/storage_api_node.rs#L12)
* Use case: [`fn storage_store_slice_u8(...)`](https://github.com/multiversx/mx-sdk-rs/blob/b0160312d19670c81b919f3029858e98f3c2228d/framework/wasm-adapter/src/api/storage_api_node.rs#L98)

Reply:
Ok, thanks, I didn't look that deeply into the code.